src/ripple/app/tests/Offer.test.cpp

```diff
@@ -149,8 +149,11 @@ class Offer_test : public beast::unit_test::suite
 
         auto const switchoverTime = STAmountCalcSwitchovers::enableUnderflowFixCloseTime ();
 
-        for (auto timeDelta : {-10s, 10s}){
-            NetClock::time_point const closeTime = switchoverTime + timeDelta;
+        for (auto timeDelta : {
+            - env.closed()->info().closeTimeResolution,
+                env.closed()->info().closeTimeResolution} )
+        {
+            auto const closeTime = switchoverTime + timeDelta;
             STAmountCalcSwitchovers switchover (closeTime);
             env.close (closeTime);
             // Will fail without the underflow fix
```

Comment:
This failure was caused by a time flag rolling over on the morning of 12/28. It is resolved in https://github.com/ximinez/rippled/commit/ce0290351498a92d2941ec3aa937e31e6c07c12b, which needs to be merged / cherry picked into this branch.